From caf9b41aa356ab5f72f09012493649f1d1db8077 Mon Sep 17 00:00:00 2001
From: Peter Jung <admin@ptr1337.dev>
Date: Thu, 5 Dec 2024 16:27:51 +0100
Subject: [PATCH 31/35] Revert "sched_ext: scx_bpf_dispatch_from_dsq_set_*()
 are allowed from unlocked context"

This reverts commit acf588f9b6fb560e986365c6b175aaf589ef1f2a.
---
 kernel/sched/ext.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/kernel/sched/ext.c b/kernel/sched/ext.c
index 166136315..ecb88c528 100644
--- a/kernel/sched/ext.c
+++ b/kernel/sched/ext.c
@@ -6473,8 +6473,6 @@ __bpf_kfunc_end_defs();
 
 BTF_KFUNCS_START(scx_kfunc_ids_unlocked)
 BTF_ID_FLAGS(func, scx_bpf_create_dsq, KF_SLEEPABLE)
-BTF_ID_FLAGS(func, scx_bpf_dispatch_from_dsq_set_slice)
-BTF_ID_FLAGS(func, scx_bpf_dispatch_from_dsq_set_vtime)
 BTF_ID_FLAGS(func, scx_bpf_dispatch_from_dsq, KF_RCU)
 BTF_ID_FLAGS(func, scx_bpf_dispatch_vtime_from_dsq, KF_RCU)
 BTF_KFUNCS_END(scx_kfunc_ids_unlocked)
-- 
2.47.1

